atomicwrites/__init__.py

```diff
@@ -131,18 +139,19 @@ def __init__(self, path, mode='w', overwrite=False):
         if 'w' not in mode:
             raise ValueError('AtomicWriters can only be written to.')
 
-        self._path = path
-        self._mode = mode
-        self._overwrite = overwrite
+        self._path = _path_to_unicode(path)  # type: Text
```

Comment:
Paths are whichever type `str` is in the respective language. Unicode paths break under Python 2 when using non-ASCII characters.